src/main/java/smile/identity/core/Signature.java

```diff
@@ -72,11 +140,12 @@ public Boolean confirm_sec_key(String timestamp, String sec_key) throws Exceptio
         } catch (Exception e) {
             throw e;
         }
-
     }
 
     private static PublicKey loadPublicKey(String apiKey) throws GeneralSecurityException, IOException {
         apiKey = apiKey.replace("-----BEGIN PUBLIC KEY-----", "").replace("-----END PUBLIC KEY-----", "").replace("\n", "").replace("\r", "").trim();
+        apiKey = apiKey.replaceAll(" ", "");
+        System.out.println("API_KEY: " + apiKey);
```

Comment:
did you mean to include this? this looks like a debug log

Reply:
Definitely a debug log; a bit of oversight on my part.